editoast/src/views/mod.rs

```diff
@@ -112,6 +116,56 @@ editoast_common::schemas! {
     work_schedules::schemas(),
 }
 
+pub type Roles = editoast_authz::roles::RoleConfig<BuiltinRole>;
+
+// This function will become a middleware once we switch to axum
```

Comment:
It might be time to address that then? 😉 [Remember](https://github.com/OpenRailAssociation/osrd/pull/8047)

Reply:
We'll do that in a follow-up, I'm opening an issue